server/controllers/profile.controller.js

```diff
@@ -131,6 +136,20 @@ class ProfileController {
     };
     return Response(res, 200, 'Successfully unfollowed user', user);
   }
+
+  static async getNotifications(req, res) {
+    const { user: { id } } = req;
+
+    const notifications = await Notification.findAndCountAll({
+      where: {
+        userId: id,
+        isSeen: false
```

Comment:
Hi @TheDrizzyWay please a little explanation on how `isSeen` works

Reply:
The idea is to check if the user has already seen or clicked on the notification. In a situation where it is `true`, it will no longer be regarded as a new notification and will not be displayed.

Reply:
Alright